packages/server/api/src/app/webhooks/webhook-simulation/webhook-simulation-controller.ts

```diff
@@ -2,7 +2,7 @@ import {
   FastifyPluginCallbackTypebox,
   Type,
 } from '@fastify/type-provider-typebox';
-import { ALL_PRINCIPAL_TYPES } from '@openops/shared';
```

Comment:
this is one of the files im not so sure about the change, from the looks of it to me this should not be a public endpoint in the first place, i can also move this to a different pr but i feel like it makes sense if one pr deals with all the public endpoints